src/main/java/budgetbuddy/ui/tab/PanelTab.java

```diff
@@ -32,4 +39,18 @@ public PanelTab(DisplayPanel panel, String imageFileName) {
             }
         });
     }
+
+    public PanelTab(DisplayPanel primaryPanel, DisplayPanel secondaryPanel, String imageFileName) {
+        this(primaryPanel, imageFileName);
+        this.secondaryPanel = secondaryPanel;
+    }
```

Comment:
This looks a little iffy, perhaps set the other constructor to call this one instead?

Reply:
Sorry, I’m not sure how that might work. How would the other constructor know whether to call this second one? It doesn’t have the secondary panel passed in.

Reply:
So the other constructor will only have `this(panel, null, imageFileName);` wheres this panel will set all 3 instance variables and also `setClosable(false)` etc.

Reply:
Ohh, so all the setting of variables are done in one constructor? Makes sense, I'll amend the code.